dist/Dockerfile

```diff
@@ -1,18 +1,10 @@
-FROM centos
+FROM centos:7
 
 RUN yum -y groupinstall 'Development Tools'
 RUN yum -y install openssl-devel
 
-ADD https://static.rust-lang.org/dist/rust-1.29.0-x86_64-unknown-linux-gnu.tar.gz rust.tar.gz
-RUN tar -xf rust.tar.gz --strip 1
-RUN ./install.sh
+ENV RUST_LOG=actix_web=error
```

Comment:
We can obsolete #18 if you add `,dkregistry=error` here

Reply:
done